server/routes/contracts/contracts.test.js

```diff
@@ -25,15 +25,14 @@ test('Save well-formed smart contract', async done => {
   done()
 })
 
-test('POST malformed smart contract', async done => {
-  await request
+test('POST malformed smart contract', done => {
+  request
     .post('/api/v0/contracts')
     .send({ wrong: 'structure' })
     .expect(400)
-    .expect({
-      error:
-        'Error: smartContract is missing or invalid. Error: network is missing or invalid. Error: abi is missing or invalid.'
-    })
+    .expect(
+      '"smartContract is missing or invalid, network is missing or invalid, abi is missing or invalid"'
```

Comment:
Should there be an "or" before "abi"?

Reply:
right now it's building this with `array.join(', ')`, so we'd have to write a decent amount more code just to add the 'or'